modules/mesh-task/outputs.tf

```diff
@@ -1,3 +1,8 @@
 output "task_definition_arn" {
   value = aws_ecs_task_definition.this.arn
 }
+
+output "task_tags" {
```

Comment:
do we need this output? Wondering how you would consume it.

Reply:
It dosen't hurt to expose it. It could be used in terratest cases to validate that the resource has the required tags.